ledger-service/http-json/src/it/edition/ce/com/daml/http/NonRepudiationTest.scala

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.http
+
+import java.util.UUID
+
+import akka.http.scaladsl.model.StatusCodes
+import com.daml.nonrepudiation.CommandIdString
+
+@SuppressWarnings(Array("org.wartremover.warts.NonUnitStatements"))
+final class NonRepudiationTest extends AbstractNonRepudiationTest {
+
+  import HttpServiceTestFixture._
+
+  "fail " in withSetup { (db, uri, encoder) =>
+    val expectedParty = "Alice"
+    val expectedNumber = "abc123"
+    val expectedCommandId = UUID.randomUUID.toString
+    val meta = Some(domain.CommandMeta(commandId = Some(domain.CommandId(expectedCommandId))))
+    val domainParty = domain.Party(expectedParty)
+    val command = accountCreateCommand(domainParty, expectedNumber).copy(meta = meta)
+    postCreateCommand(command, encoder, uri)
+      .flatMap { case (status, _) =>
+        status shouldBe StatusCodes.InternalServerError
```

Comment:
This is not what we actually want to happen, I'd say 403 (Forbidden) makes more sense. Working on this.

Reply:
I think InternalServerError is fine for now. There is a broader topic here on the JSON API reporting any failure coming from the ledger as 500 but I wouldn’t address this specifically for non-repudiation.

Reply:
Ok, thanks.